src/pages/Dashboard/components/SummaryChart.tsx

```diff
@@ -23,14 +24,31 @@ interface Country {
 
 interface SummaryChartProps {
   data: Country[];
-  filter: "sex" | "lgbtq";
+  filter: "sex" | "lgbtq" | "result";
   w: number
   h: number
+  isSummaryPage: boolean
+  axisOption?: string
 }
 
-export const SummaryChart = ({ data, filter, w, h }: SummaryChartProps) => {
+export const SummaryChart = ({ data, filter, w, h, isSummaryPage, axisOption }: SummaryChartProps) => {
   const { t } = useTranslation();
 
+  let leftMargin = 0;
+
+  // The y-axis label needs more space to be displayed if we are on summary
+  if (isSummaryPage) { leftMargin = 38 };
+
+  // NOTE: this is just a test to see if changing the drop down to a different value actually affects the chart
+  if (axisOption === "gender") { w = 500; h = 200 }
```

Comment:
if this is a test do we need to merge it in or can we delete it?

Reply:
It will be deleted eventually, I just wanted to see if anything happened. We can get rid of it before it's merged